### DISCUSSION TITLE
Enable browser cache on static pages/components

### DISCUSSION THREAD

**Discussion by elyesnaouar** [PrestaShop/PrestaShop#34481] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Motivation: Browser caching speeds up the display of your web pages, thanks to a technique that consists in storing a local capture of the page so that it can be used for display instead of having to be generated anew on each visit. It is recommended to use the cache on certain pages by removing the "no-cache" values from the http header "Cache-Control".

### Steps to reproduce

installe a shop using v8.1.2

### PrestaShop version(s) where the bug happened

8.1.2

**Comment by Hlavtox** [comment]
Nearly impossible to do in dynamic ecommerce environment. There are modules for this, but it's problematic. It reloads some parts of the website with ajax requests and brings more issues than it solves.

@PrestaShop/qa-functional Please convert it to discussion. :-)

**Comment by AureRita** [comment]
Hi @elyesnaouar 

Thank you for your report. Indeed, as @Hlavtox said, this one could be considered as a new feature that we would like to implement, so I will convert this one into a discussion. 
FYI @PrestaShop/quality-council about this feature.

Thank you

**Reply by fabienvallon** [reply]
Yes thx
